### PR TITLE
feat: Added profile and affiliation step to new form

### LINKED ISSUE #564: Landscape Profile: New Landscape form multistep support
## Description
- We should be able to easily add or remove steps from the landscape creation form
- Handle "save for now" option

### PR DESCRIPTION
**Pull request by josebui**
### Checklist
- [ ] Corresponding issue has been opened
- [ ] New tests added
- [ ] English strings reviewed and copyedited
- [ ] Strings are localized
- [ ] Verified on mobile
- [ ] Verified on desktop
- [ ] Verified accessibility ([Accessibility - Web applications](https://docs.google.com/document/d/1VwPDyLqw7r_iISMgDZr1FdAmAqqOIIVIxQqsYA-HaHE/edit?usp=sharing))

### Related Issues
Fixes #564

### CODE REVIEW

File: src/landscape/components/LandscapeForm/AffiliationStep.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        {t('landscape.form_cancel')}
+      </Button>
+    </Stack>
+  );
+};
+
+const AffiliationStep = props => {
+  const { t } = useTranslation();
+  const { setUpdatedLandscape, landscape, isNew, onCancel } = props;
+  console.log({ setUpdatedLandscape, landscape, isNew });
```
Comment (david-code) [inline src/landscape/components/LandscapeForm/AffiliationStep.js:46]: chore: get rid of console.log
```suggestion
```

**Review by david-code — CHANGES_REQUESTED** [review]

File: src/landscape/components/LandscapeForm/InfoStep.js [hunk 2]
```diff
@@ -97,9 +97,7 @@ const CountrySelector = props => {
 const InfoStep = props => {
   const { t } = useTranslation();
   const navigate = useNavigate();
-  const { slug } = useParams();
-  const isNew = !slug;
-  const { setUpdatedLandscape, landscape } = props;
+  const { setUpdatedLandscape, landscape, isNew } = props;
```
[the remark this answers is not in the document]
Comment (josebui) [reply src/landscape/components/LandscapeForm/InfoStep.js:100]: @david-code In that case `isNew` will be falsy, so not really React magic more a Javascript thing

File: src/landscape/components/LandscapeForm/ProfileStep.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        {t('landscape.form_cancel')}
+      </Button>
+    </Stack>
+  );
+};
+
+const ProfileStep = props => {
+  const { t } = useTranslation();
+  const { setUpdatedLandscape, landscape, isNew, onCancel } = props;
+  console.log({ setUpdatedLandscape, landscape, isNew });
```
[the remark this answers is not in the document]
Comment (josebui) [reply src/landscape/components/LandscapeForm/ProfileStep.js:46]: @david-code This Step needs to be finished as part of other Issues what is why this temporary console.log is here

File: src/landscape/components/LandscapeForm/AffiliationStep.js (hunk 1 above)
Comment (josebui) [reply src/landscape/components/LandscapeForm/AffiliationStep.js:46]: @david-code same here

File: src/landscape/components/LandscapeForm/ProfileStep.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  return (
+    <Stack
+      direction={{ xs: 'column', sm: 'row' }}
+      justifyContent="space-between"
+    >
+      <Button variant="contained">{t('landscape.form_update')}</Button>
+      <Button variant="text" onClick={onCancel}>
+        {t('landscape.form_cancel')}
+      </Button>
+    </Stack>
```
Comment (paulschreiber) [inline src/landscape/components/LandscapeForm/ProfileStep.js:39]: Suggestion. This looks very similar to what's in `AffiliationStep.js`. Make this into a component that you reuse.

**Review by paulschreiber — CHANGES_REQUESTED** [review]

**Review by paulschreiber — CHANGES_REQUESTED** [review]
This looks pretty good.

issue: please fix the tests so they pass.

**Comment by josebui** [comment]
@paulschreiber This are intermidiate steps for the complete feature, I will not be fixing tests at this stage since a lot of things might change after

**Review by paulschreiber — APPROVED** [review]

**Pull request merged**